ui/src/components/Navbar.svelte

```diff
@@ -101,8 +104,23 @@
           </div>
           {#if notificationsOpen}
             <div
-              class="absolute top-10 w-max flex flex-col gap-4 bg-white dark:bg-black rounded-xl border border-white overflow-hidden"
+              class="absolute top-10 w-max p-3 flex flex-col gap-4 bg-white dark:bg-black rounded-xl border border-grey overflow-hidden"
             >
+              <div class="flex justify-between">
+                <div class="text-xl">Notifications</div>
+                <div class="relative flex items-center justify-end">
+                  <div class="relative">
+                    <button on:click|stopPropagation={toggleMuteNotifications} class="switch block border border-black dark:border-white w-14 h-8 rounded-full flex justify-between items-center cursor-pointer">
+                      <BellIcon
+                        class={`p-[3px] transform translate-x-[3px] ${!$state.muteNotifications ? 'text-white dark:text-black bg-black dark:bg-white rounded-full' : ''}`}
+                      />
+                      <MutedIcon
+                        class={`p-[3px] transform -translate-x-[3px] ${$state.muteNotifications ? 'text-white dark:text-black bg-black dark:bg-white rounded-full' : ''}`}
+                      />
```

Comment:
In theory you can put these classes on a wrapping `<div></div>` so that you don't have to modify the .svg file to deal with `$$props.class` which is a variable unique to svelte.